Packs/EDL/ReleaseNotes/3_2_13.md

```diff
@@ -0,0 +1,9 @@
+
+#### Integrations
+
+##### Generic Export Indicators Service
+
+- Undeprecated the *Legacy Query* parameter.
+- Removed *Listen Port* and *credentials* parameters from cortex XSIAM.
+- Move out from `Advanced settings` the *query* parameter.
```

Comment:
I am not sure what this means?  Is it no longer part of the Advanced settings but it is part of the regular settings?  Was it just removed?

Reply:
It means that the `Indicator Query` params was inside the Advanced settings and now we move it out from there to `Collect` section

<img width="582" alt="image" src="https://github.com/demisto/content/assets/46249224/37fd0e09-2ecf-4e76-9d78-665fac4ea72f">